src/components/reusable/timepicker/timepicker.scss

```diff
@@ -0,0 +1,160 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+}
+
+.time-picker {
+  width: 147px;
```

Comment:
This can be removed, don't want fixed widths anywhere really.

Reply:
yes kyn-text-input also has same issue.

Reply:
Yeah, would be great if you could make that fix for both, thanks

Reply:
sure thing

Reply:
Done at timepicker.scss , textinput.scss and textarea.scss